indico/modules/attachments/templates/_management_info_column.html

```diff
@@ -7,7 +7,8 @@
                data-locator="{{ obj.locator|tojson|forceescape }}"
                data-attachment-editor>
                 {% if not obj.attachment_count %}
-                    <em>{% trans %}None{% endtrans %}</em>
+                    {# i18n: No attachments #}
+                    <em>{% trans "Attachments" %}None{% endtrans %}</em>
```

Comment:
```suggestion
                    <em>{% trans 'Attachments' %}None{% endtrans %}</em>
```

I'd prefer single quotes. This also allows using the same style anywhere (e.g. inside HTML attributes where double quotes would technically work but might mess w/ syntax highlighting)

Reply:
Changed :+1: